pkg/cli/cmd/radinit/aws.go

```diff
@@ -53,12 +54,24 @@ func (r *Runner) enterAWSCloudProvider(ctx context.Context) (*aws.Provider, erro
 		return nil, err
 	}
 
-	accountId, err := r.getAccountId(ctx, accessKeyID, secretAccessKey)
+	accountId, err := r.getAccountId(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	region, err := r.selectAWSRegion(ctx, QueryRegion, accessKeyID, secretAccessKey)
+	addingCloudProvider, err := prompt.YesOrNoPrompt(fmt.Sprintf(confirmAWSAccountIDPromptFmt, accountId), prompt.ConfirmYes, r.Prompter)
+	if err != nil {
+		return nil, err
```

Comment:
what would cause this to error out?

Reply:
error could be caused when creating a new Program in GetListInput function. https://github.com/radius-project/radius/blob/c62434ff63af925dabfa36f6a9a781454fcc322f/pkg/cli/prompt/prompt.go#L74-L77